.github/sync-files.yaml

```
@@ -26,6 +26,11 @@
         '      - "**/*.jpg"
               - "**/*.py"' \
         {source}
+        sd -- \
+        '          latest: (.*)'\
+        '          latest: $1
+          mkdocs-requirements-txt: ./mkdocs-requirements.txt'
+        {source}
```

It looks `\` is missing.
```suggestion
        sd -- \
        '          latest: (.*)'\
        '          latest: $1
          mkdocs-requirements-txt: ./mkdocs-requirements.txt' \
        {source}
```